mitmproxy/tools/console/master.py

```diff
@@ -123,12 +123,30 @@ def get_editor(self) -> str:
         else:
             return "vi"
 
+    def get_hex_editor(self) -> str:
+        if m := os.environ.get("MITMPROXY_EDITOR"):
+            return m
+        if m := os.environ.get("EDITOR"):
+            return m
+        for editor in "hexedit", "hexyl":
+            if shutil.which(editor):
+                return editor
+        if os.name == "xxd":
```

Comment:
I don't understand this comparison. Can you explain?

Reply:
I figured out the error :).
I will push the new code.